datacube_apps/stacker/fixer.py

```diff
@@ -0,0 +1,273 @@
+"""
+Create time-stacked NetCDF files
```

Comment:
This fixes up metadata doesn't it?

It's nice to include some background information too, about why this is a useful tool.